mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule KafkaEx.Mixfile do
   def project do
     [
       app: :kafka_ex,
-      version: "0.8.1",
+      version: "0.8.2",
```

Comment:
The version bump is handled when we do a release, so this part should be removed.